va/va_dec_av1.h

```diff
@@ -0,0 +1,546 @@
+     *  except for the key frame by checking ref_frame_id[]. If missing frame
+     *  is identified, application may choose to perform error recovery by
+     *  pointing problematic index to an alternative frame buffer.
+     *  Driver is not responsible to validate reference frames' id.
+     */
+    VASurfaceID             ref_frame_map[8];
+
+     /** \brief Reference frame indices.
+     *
+     *  Contains a list of indices into ref_frame_map[8].
+     *  It specifies the reference frame correspondence.
+     *  The indices of the array are defined as [LAST_FRAME – LAST_FRAME,
+     *  LAST2_FRAME – LAST_FRAME, …, ALTREF2_FRAME – LAST_FRAME], where each
+     *  symbol is defined as:
+     *  enum{INTRA_FRAME = 0, LAST_FRAME, LAST2_FRAME, LAST3_FRAME, GOLDEN_FRAME,
+     *  BWDREF_FRAME, ALTREF_FRAME};
+     */
+    uint8_t                 ref_frame_idx[7];
+
+    /** \brief primary reference frame index
+     *  Index into ref_frame_idx[], specifying which reference frame contains
+     *  propagated info that should be loaded at the start of the frame.
+     *  When value equals PRIMARY_REF_NONE (7), it indicates there is
+     *  no primary reference frame.
+     *  value range [0..7]
+     */
+    uint8_t                 primary_ref_frame;
+
+    uint8_t                 order_hint;
+
+    VASegmentationStructAV1 seg_info;
+    VAFilmGrainStructAV1    film_grain_info;
+
+    /** \brief tile structure
+     *  When uniform_tile_spacing_flag == 1, the value of tile_cols and
+     *  tile_rows need to be power of 2, and width_in_sbs_minus_1[] and
+     *  height_in_sbs_minus_1[] should be ignored, which will be generated
+     *  by driver based on tile_cols and tile_rows.
+     */
+    uint8_t                 tile_cols;
+    uint8_t                 tile_rows;
+    uint16_t                width_in_sbs_minus_1[63];
+    uint16_t                height_in_sbs_minus_1[63];
+
+    /** \brief number of tiles in large scale tile list
+     *  Valid only when large_scale_tiles == 1.
+     */
+    uint16_t                tile_count_minus_1;
+
+    /* specify the tile index for context updating */
+    uint16_t                context_update_tile_id;
+
+    union
+    {
+        struct
+        {
+            /** \brief flags for current picture
+             *  same syntax and semantic as those in AV1 code
+             */
+
+             /** \brief Frame Type
+              *  0:     KEY_FRAME;
+              *  1:     INTER_FRAME;
+              *  2:     INTRA_ONLY_FRAME;
+              *  3:     SWITCH_FRAME
+              *  For SWITCH_FRAME, application shall set error_resilient_mode = 1,
+              *  refresh_frame_flags, etc. appropriately. And driver will convert it
+              *  to INTER_FRAME.
+              */
+            uint32_t        frame_type                                  : 2;
+            uint32_t        show_frame                                  : 1;

```

Comment:
why you need show_frame and showable_frame? The frame reordering and output should handle by up layer.

Reply:
@ZefuLI, any feedback for this?

Reply:
I will remove show_frame and showable_frame.